site/app/templates/HomePage.twig

```diff
@@ -56,15 +56,21 @@
                     </a>
                 </li>
             {% endif %}
-
-            <li>
-              <br>
-              <select id="theme_change_select" class="form-control" aria-label="Change Submitty's Theme" onchange="updateTheme();">
-                <option value="system">Follow Device Theme</option>
-                <option value="system_black">Follow Device Theme (Black)</option>
-                <option value="light">Light Mode</option>
-                <option value="dark">Dark Mode</option>
-                <option value="dark_black">Dark Mode (Black)</option>
+            <li class="li_with_drop_down">
```

Comment:
We're trying to get part 1 of this feature released for the first day of summer classes so instructors can collect this data from students.

We will soon progress to part 2 - allowing instructors to view & download this information

and part 3 - modifying the displayed due date/time for homework 

(& eventually other relevant places on the site)

Reply:
Resolving after @bmcutler explanation.

Reply:
We should probably put on the site itself (and not just a link to documentation) that setting this will not affect any pages, and that all dates will continue to use the server time for deadlines, and such. Unless we're positive 3 is going to be done before any deadlines roll up.

Reply:
Classes start on Tuesday
Yes, we expect to have date modification done very soon.

Reply:
FYI:  We currently display the server timezone next to the gradeable deadlines.